src/commonTest/kotlin/ExampleSpec.kt

```diff
@@ -0,0 +1,12 @@
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class ExampleSpec : StringSpec({
+  "true shouldBe true" {
+    true shouldBe true
+  }
+
+  "exception should fail" {
+//    throw RuntimeException("Boom2!")
```

Comment:
what's the plan here?

Reply:
Uhm, it served as an example of a failing test to try out the different test flavors but it doesn't really makes sense to stay in the template.